src/Symfony/Component/Clock/Tests/DatePointTest.php

```diff
@@ -45,6 +45,41 @@ public function testCreateFromFormat()
         DatePoint::createFromFormat('Y-m-d H:i:s', 'Bad Date');
     }
 
+    /** @dataProvider provideValidTimestamps */
```

Comment:
we have very few oneliners for dataproviders in the codebase. I think we prefer multi lines for docblocks:
```suggestion
    /**
     * @dataProvider provideValidTimestamps
     */
```